.travis/script.sh

```diff
@@ -8,5 +8,7 @@ make install-coverage
 
 python -m pytest -v -x --cov PIL --cov-report term Tests
 
+pushd /tmp/check-manifest && check-manifest --ignore ".coveragerc,.editorconfig,*.yml,*.yaml" && popd
```

Comment:
I resolved the merge conflict using the UI, but it kept this line that was removed in https://github.com/python-pillow/Pillow/pull/3940. Please can you update the PR?